doc/releases/release-notes-4.1.rst

```diff
@@ -243,6 +243,18 @@ Libraries / Subsystems
 
 * Crypto
 
+  * The Kconfig symbol :kconfig:option:`CONFIG_MBEDTLS_PSA_STATIC_KEY_SLOTS` was
+    added to allow Mbed TLS to use pre-allocated static buffers to store key material
+    in its PSA Crypto core instead of heap-allocated ones. This can help reducing
+    (or removing, if no other component makes use of it) heap memory requirements
```

Comment:
```suggestion
    in its PSA Crypto core instead of heap-allocated ones. This can help reduce
    (or remove, if no other component makes use of it) heap memory requirements
```

Reply:
`reducing` => `reduce` remains